content/en/real_user_monitoring/browser/setup/server.md

```diff
@@ -32,21 +42,79 @@ The automatic installation method requires that you have the [Datadog Agent][2]
 <div class="alert alert-warning">To request support for a web server that is not listed here, <a href="https://www.datadoghq.com/private-beta/rum-sdk-auto-injection/">fill out this form.</a></div>
 
 {{< tabs >}}
-{{% tab "Nginx" %}}
+{{% tab "NGINX" %}}
+
+The Auto-Instrumentation method leverages the [NGINX Dynamic Modules capability][1] to implement a response body filter. The filter injects the RUM SDK into the response body for responses 
+identified as HTML. You can instrument your RUM application automatically or manually.
+
+[1]: https://docs.nginx.com/nginx/admin-guide/dynamic-modules/dynamic-modules/
+
 
-The Auto-Instrumentation method leverages the [Nginx Dynamic Modules capability][1] to implement a response body filter. The filter injects the RUM SDK into the response body for responses identified as HTML.
+{{% collapse-content title="Automatic instrumentation (recommended)" level="h5" %}}
 
 To automatically instrument your RUM application:
 
-1. In Datadog, navigate to the [**Digital Experience > Add an Application Page**][2] and select the JavaScript (JS) application type.
-2. Select **Auto-Instrumentation** and **Nginx**.
-3. Set your Session and Session Replay sample rates. See [guidance on configuring sampling][3].
-4. Copy and run the installer command to load the Datadog RUM SDK Injector onto your Nginx module.
-5. After the installer successfully installs the SDK Injector, restart Nginx to begin collecting RUM sessions.
+1. In Datadog, navigate to the [**Digital Experience > Add an Application Page**][1] and select the JavaScript (JS) application type.
+2. Select **Auto-Instrumentation** and **NGINX**.
+3. Set your Session and Session Replay sample rates. See [guidance on configuring sampling][2].
+4. Copy and run the installer command to load the Datadog NGINX Module with the RUM SDK Injector onto NGINX.
+5. After the installer successfully installs the SDK Injector, restart NGINX to begin collecting RUM sessions.
+6. (Optional) To verify the module is successfully injecting the RUM Browser SDK into HTML pages, check the NGINX error logs for relevant messages. The module logs important steps during the injection process. Ensure that NGINX is configured with at least the `INFO` log level with the following:
 
-[1]: https://docs.nginx.com/nginx/admin-guide/dynamic-modules/dynamic-modules/
-[2]: https://app.datadoghq.com/rum/list
-[3]: /real_user_monitoring/guide/sampling-browser-plans/
+   ```javascript
+   error_log <file> info;
+   ```
+
+[1]: https://app.datadoghq.com/rum/list
+[2]: /real_user_monitoring/guide/sampling-browser-plans/
+
+{{% /collapse-content %}}
+
+{{% collapse-content title="Manual instrumentation" level="h5" %}}
```

Comment:
**💭 thought:** ‏it's a bit confusing to talk about "Manual instrumentation" on a page called "Auto instrumentation". Maybe we should have another wording, like: `Using the installer` and `Manual configuration`

Reply:
Finally! Thank you @BenoitZugmeyer